modules/ldsg.js

```diff
@@ -34,7 +33,9 @@ const sendOptions = [
 const replyOption = [
   u.MessageActionRow().setComponents([
     new u.Button().setCustomId("suggestionReply").setEmoji("🗨️").setLabel("Reply to user").setStyle(Discord.ButtonStyle.Primary),
-    new u.Button().setCustomId("suggestionRename").setEmoji("✏️").setLabel("Edit title").setStyle(Discord.ButtonStyle.Primary)
+    new u.Button().setCustomId("suggestionRename").setEmoji("✏️").setLabel("Set title").setStyle(Discord.ButtonStyle.Primary),
+    new u.Button().setCustomId("suggestionIssue").setEmoji("✏️").setLabel("Set Issue").setStyle(Discord.ButtonStyle.Primary),
+    new u.Button().setCustomId("suggestionCause").setEmoji("✏️").setLabel("Set Root Cause").setStyle(Discord.ButtonStyle.Primary)
```

Comment:
lets just have one "edit" button. You can have 5 action rows in a modal with 1 text input each.

```js
const title = modal.getTextInputValue("suggestionTitle")
const issue = modal.getTextInputValue("suggestionIssue")
const cause = modal.getTextInputValue("suggestionCause")
if (!title && !issue && !cause) return modal.reply({ content: "I need some stuff to change!", ephemeral: true })
const fields = embed.fields
if (title) post.setTitle(title)
if (issue) embed.setFields([...fields.filter(f => f.name != "Issue"), { name: "Issue", value: issue }])
if (cause) embed.setFields([...fields.filter(f => f.name != "Root Cause"), { name: "Root Cause", value: cause }])
```